backend/routes/api/index.js

```diff
@@ -4,14 +4,13 @@ const sessionRouter = require("./session.js");
 const usersRouter = require("./users.js");
 const myRouter = require("./my");
 const songRouter = require("./songs");
+const artistRouter = require("./artist");
```

Comment:
Should give this the '/artists' argument so you dont need to preface all your routes with that in the artists router.